lib/mongoid/contextual/map_reduce.rb

```diff
@@ -299,7 +299,7 @@ def output_collection
       # @since 3.0.0
       def results
```

Comment:
should be called result instead of results just because is returning one element, the same as in geo_near.rb